Model/Transaction/Refund.php

```diff
@@ -77,9 +77,26 @@ public function build(
         );
 
         if (isset($this->request['line_items'])) {
-            foreach ($this->request['line_items'] as $lineItem) {
+            $adjustmentFee = $creditmemo->getAdjustmentNegative();
+            $adjustmentRefund = $creditmemo->getAdjustmentPositive();
+
+            foreach ($this->request['line_items'] as $k => $lineItem) {
+                $lineItemSubtotal = $lineItem['unit_price'] * $lineItem['quantity'];
+                $this->request['line_items'][$k]['discount'] += ($adjustmentFee * ($lineItemSubtotal / $creditmemo->getSubtotal()));
```

Comment:
@ryanreeves-taxjar Since we call `$creditmemo->getSubtotal()` at the top of the `build` method we could just use `$subtotal` here right?